demo/demo.js

```diff
@@ -2617,5 +2617,63 @@ var demos = {
 				}
 			}
 		}
+	},
+	Customization: {
+		RectangleForPoints: {
+			options: {
+				data: {
+					columns: [
+						['data1', 100, 200, 1000, 900, 500],
+						['data2', 20, 40, 500, 300, 200]
+					]
+				},
+				point: {
+					type: "rectangle"
+				}
+			}
+		},
+		TriangleForPoints: {
+			options: {
+				data: {
+					columns: [
+						['data1', 100, 200, 1000, 900, 500],
+						['data2', 20, 40, 500, 300, 200]
+					]
+				},
+				point: {
+					type: "custom",
+
+					create(element, cssClassFn, sizeFn, fillStyle) {
```

Comment:
How about integrate this as "triangle" type? That will be really useful.

Reply:
I've intergrated this "triangle" type, however I probably need your help with the `update` methods when `withTransition` and `flow` arguments are used.

Reply:
Got it @julien. I think this PR is great, but we need to have a detailed look on backward compatibility with other options without confusion.

Reply:
@netil thanks for the feedback. I also just updated the demo.